federation/README.md

```diff
@@ -1,26 +1,32 @@
-### Follow these steps to get started using Spice to federate SQL queries across data sources.
+# Federated SQL Query
 
-**Step 1.** Clone the quickstarts repo locally and navigate to the `federation` directory.
+Fetch combined data from S3 Parquet, PostgreSQL, and Dremio in a single query.
+
+## Requirements
+
+- Docker
+
```

Comment:
Need a header between `- Docker` and `**Step 1.**` otherwise the two lists merge.